.travis.yml

```diff
@@ -51,9 +57,9 @@ before_install:
     # Builds Complete
     - popd
     # pip installs
-    - pip install psutil requests jsonschema strict-rfc3339 xvfbwrapper pytest funcparserlib
-    - pip install git+https://github.com/sunu/qt5reactor
-    - "if [[ $SYSTEM_PACKAGES == 'false' ]]; then pip install -UI Pillow twisted service_identity lupa; fi"
+    - pip install psutil requests jsonschema strict-rfc3339 xvfbwrapper pytest funcparserlib qt5reactor-fork six
+    - if [[ $SYSTEM_PACKAGES == 'false' ]]; then pip install -UI Pillow service_identity lupa; fi
+    - if [[ $TRAVIS_PYTHON_VERSION == 3.4 ]]; then git clone https://github.com/twisted/twisted; pushd twisted; python setup.py install; popd; else pip install twisted; fi
```

Comment:
`pip install https://github.com/twisted/twisted/archive/trunk.zip#egg=twisted` is better because it doesn't even require git. Anyways, they released twisted 15.3.0 yesterday